MeshLib/MeshSurfaceExtraction.h

```diff
@@ -32,8 +32,22 @@ class Node;
 class MeshSurfaceExtraction
 {
 public:
-    /// Returns a vector of the areas assigned to each node on a surface mesh.
-    static std::vector<double> getSurfaceAreaForNodes(const MeshLib::Mesh &mesh);
+    /// Returns a vector of values where each value is associated with a
+    /// particular node. Since a node is part of elements, it is possible to
+    /// assign an area per element to this node. Each value of the return vector
+    /// is the sum of the assigned area (per element) multiplied by the property
+    /// value of the element.
+    /// @param mesh a surface mesh containing a property \c prop_name assigned
+    /// to cells
+    /// @param prop_name name of the cell based property within the \c mesh
+    /// @return vector of integration value associated to the surface mesh nodes
+    static std::vector<double> getSurfaceIntegratedValueForNodes(
```

Comment:
I'm also unsure. it could be placed under
- MeshLib/MeshEditing or new dir Utils (?)
- new dir NumLib/Integration
- ProcessLib/Utils

`NumLib/Fem/Integration` isn't a good place for this because it focuses on element-level integration.